### ISSUE TITLE
Python 2 is EOL

### ISSUE THREAD

**Issue by DuBistKomisch** [cyxx/extract_android_ota_payload#9]
```
 $ python ~/src/extract_android_ota_payload/extract_android_ota_payload.py payload.bin payload
Traceback (most recent call last):
  File "/home/jake/src/extract_android_ota_payload/extract_android_ota_payload.py", line 14, in <module>
    raise Exception("Python 2.x is required")
Exception: Python 2.x is required
```

and warning from `pip2 install -r requirements.txt`

>DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7

**Comment by dhananjaygbhardwaj** [comment]
On macOS 10.15 (python3, pip3, and protobuf installed)

Used: `python3 extract_android_ota_payload.py payload.bin`

Got the following:

```
dhananjay@Dhananjays-MacBook-Air extract_android_ota_payload-master % python3 extract_android_ota_payload.py payload.bin
Traceback (most recent call last):
  File "extract_android_ota_payload.py", line 14, in <module>
    raise Exception("Python 2.x is required")
Exception: Python 2.x is required
```

**Comment by cyxx** [comment]
Changes for python 3 have been merged (#12).

The script also require having 'bzcat' and 'xzcat' in PATH.
